Altis_Life.Altis/core/actions/fn_impoundAction.sqf

```diff
@@ -77,4 +109,4 @@ if (count crew _vehicle isEqualTo 0) then {
     hint localize "STR_NOTF_ImpoundingCancelled";
 };
 
-life_action_inUse = false;
+life_action_inUse = false;
```

Comment:
Changes removed, as cops will never meet a surplus vehicle which has not been "disabled" by the spawnVehicle script.